greenheart/tools/eco/utilities.py

```diff
@@ -1444,6 +1442,1057 @@ def save_energy_flows(
+    if grid_case in ("grid-only", "hybrid-grid"):
+        electrolyzer_grid_profile_data_dict = {'Energy to electrolyzer from grid (kWh)': annual_energy_to_electrolyzer_from_grid}
+        electrolyzer_grid_profile_df = pd.DataFrame(data=electrolyzer_grid_profile_data_dict)
+        electrolyzer_grid_profile_df = electrolyzer_grid_profile_df.reset_index().rename(columns={'index':'Interval'})
+        electrolyzer_grid_profile_df['Interval'] = electrolyzer_grid_profile_df['Interval']+1
+        electrolyzer_grid_profile_df = electrolyzer_grid_profile_df.set_index('Interval')
+
+    # Instantiate object to hold EI values per year
+    electrolysis_Scope3_EI = np.nan
+    electrolysis_Scope2_EI = np.nan
+    electrolysis_Scope1_EI = np.nan
+    electrolysis_total_EI  = np.nan
+    smr_Scope3_EI = np.nan
+    smr_Scope2_EI = np.nan
+    smr_Scope1_EI = np.nan
+    smr_total_EI  = np.nan
+    smr_ccs_Scope3_EI = np.nan
+    smr_ccs_Scope2_EI = np.nan
+    smr_ccs_Scope1_EI = np.nan
+    smr_ccs_total_EI  = np.nan
+    atr_Scope3_EI = np.nan
+    atr_Scope2_EI = np.nan
+    atr_Scope1_EI = np.nan
+    atr_total_EI  = np.nan
+    atr_ccs_Scope3_EI = np.nan
+    atr_ccs_Scope2_EI = np.nan
+    atr_ccs_Scope1_EI = np.nan
+    atr_ccs_total_EI  = np.nan
+    NH3_electrolysis_Scope3_EI = np.nan
+    NH3_electrolysis_Scope2_EI = np.nan
+    NH3_electrolysis_Scope1_EI = np.nan
+    NH3_electrolysis_total_EI  = np.nan
+    NH3_smr_Scope3_EI = np.nan
+    NH3_smr_Scope2_EI = np.nan
+    NH3_smr_Scope1_EI = np.nan
+    NH3_smr_total_EI  = np.nan
+    NH3_smr_ccs_Scope3_EI = np.nan
+    NH3_smr_ccs_Scope2_EI = np.nan
+    NH3_smr_ccs_Scope1_EI = np.nan
+    NH3_smr_ccs_total_EI  = np.nan
+    NH3_atr_Scope3_EI = np.nan
+    NH3_atr_Scope2_EI = np.nan
+    NH3_atr_Scope1_EI = np.nan
+    NH3_atr_total_EI  = np.nan
+    NH3_atr_ccs_Scope3_EI = np.nan
+    NH3_atr_ccs_Scope2_EI = np.nan
+    NH3_atr_ccs_Scope1_EI = np.nan
+    NH3_atr_ccs_total_EI  = np.nan
+    steel_electrolysis_Scope3_EI = np.nan
+    steel_electrolysis_Scope2_EI = np.nan
+    steel_electrolysis_Scope1_EI = np.nan
+    steel_electrolysis_total_EI  = np.nan
+    steel_smr_Scope3_EI = np.nan
+    steel_smr_Scope2_EI = np.nan
+    steel_smr_Scope1_EI = np.nan
+    steel_smr_total_EI  = np.nan
+    steel_smr_ccs_Scope3_EI = np.nan
+    steel_smr_ccs_Scope2_EI = np.nan
+    steel_smr_ccs_Scope1_EI = np.nan
+    steel_smr_ccs_total_EI  = np.nan
+    steel_atr_Scope3_EI = np.nan
+    steel_atr_Scope2_EI = np.nan
+    steel_atr_Scope1_EI = np.nan
+    steel_atr_total_EI  = np.nan
+    steel_atr_ccs_Scope3_EI = np.nan
+    steel_atr_ccs_Scope2_EI = np.nan
+    steel_atr_ccs_Scope1_EI = np.nan
+    steel_atr_ccs_total_EI  = np.nan
+
+    # Instantiate lists to hold data for all LCA calculations for all cambium years
+    electrolysis_Scope3_emission_intensity = []
+    electrolysis_Scope2_emission_intensity = []
+    electrolysis_Scope1_emission_intensity = []
+    electrolysis_emission_intensity = []
```

Comment:
I think I'd like it better if we call it `emission_total` rather than just `emission` to follow the same format above